com.Adobe.Flash-Player-Projector.json

```diff
@@ -0,0 +1,64 @@
+{
+    "app-id" : "com.Adobe.Flash-Player-Projector",
+    "runtime" : "org.freedesktop.Platform",
+    "runtime-version" : "1.6",
```

Comment:
`18.08`

Reply:
I've switched to using the GNOME platform and SDK, as the app needs GTK+ 2.x to run.